app/request_lookup.py

```diff
@@ -0,0 +1,28 @@
+import json
+import os
+import requests
+from app import settings
+from structlog import get_logger
+
+logger = get_logger()
+request_timeout = 30
+
+def get_results(type, search_value):
+    request_string = os.path.join(settings.LOOKUP_URL, '_all', type, '_search?q=' + search_value)
```

Comment:
I know this is super simple, but is it worth using the python es library?  How are we authenticating?